src/main/java/org/jabref/gui/entryeditor/CitationRelationsTab.java

```diff
@@ -0,0 +1,628 @@
+
+        styleFetchedListView(citingListView);
+        styleFetchedListView(citedByListView);
+
+        return container;
+    }
+
+    private void styleFetchedListView(CheckListView<CitationRelationItem> listView) {
+        PseudoClass entrySelected = PseudoClass.getPseudoClass("entry-selected");
+        new ViewModelListCellFactory<CitationRelationItem>()
+                .withGraphic(e -> {
+
+                    HBox separator = new HBox();
+                    HBox.setHgrow(separator, Priority.SOMETIMES);
+                    Node entryNode = getEntryNode(e.getBibEntry());
+                    HBox.setHgrow(entryNode, Priority.ALWAYS);
+                    HBox hContainer = new HBox();
+                    hContainer.prefWidthProperty().bind(listView.widthProperty().subtract(25));
+
+                    if (e.isLocal()) {
+                        Button jumpTo = IconTheme.JabRefIcons.LINK.asButton();
+
+                        jumpTo.getStyleClass().add("addEntryButton");
+                        jumpTo.setOnMouseClicked(event -> {
+                            libraryTab.showAndEdit(e.getBibEntry());
+                            libraryTab.clearAndSelect(e.getBibEntry());
+                            citingTask.cancel();
+                            citedByTask.cancel();
+                        });
+                        hContainer.getChildren().addAll(entryNode, separator, jumpTo);
+                    } else {
+                        ToggleButton addToggle = IconTheme.JabRefIcons.ADD.asToggleButton();
+                        EasyBind.subscribe(addToggle.selectedProperty(), selected -> {
+                            if (selected) {
+                                addToggle.setGraphic(IconTheme.JabRefIcons.ADD_FILLED.withColor(IconTheme.SELECTED_COLOR).getGraphicNode());
+                            } else {
+                                addToggle.setGraphic(IconTheme.JabRefIcons.ADD.getGraphicNode());
+                            }
+                        });
+                        addToggle.getStyleClass().add("addEntryButton");
+                        addToggle.selectedProperty().bindBidirectional(listView.getItemBooleanProperty(e));
+                        hContainer.getChildren().addAll(entryNode, separator, addToggle);
+                    }
+                    hContainer.getStyleClass().add("entry-container");
+
+                    /*if (citingListView.getItems().size() == 1) {
+                        selectAllNewEntries(listView);
+                    }*/
+
+                    return hContainer;
+                })
+                .withOnMouseClickedEvent((ee, event) -> listView.getCheckModel().toggleCheckState(ee))
+                .withPseudoClass(entrySelected, listView::getItemBooleanProperty)
+                .install(listView);
+
+        listView.setSelectionModel(new NoSelectionModel<>());
+    }
+
+    private Node getEntryNode(BibEntry entry) {
```

Comment:
Please create a new class responsible for displaying an entry (e.g. BibEntryView), and unify the display with https://github.com/JabRef/jabref/blob/eb5c8ca33a608a4a20b614d8e3bdc4bc17413b3e/src/main/java/org/jabref/gui/importer/ImportEntriesDialog.java#L153

Reply:
We created the class BibEntryView to unify the two list views. Any further suggestions?